data_sources.py

```diff
@@ -5,4 +5,8 @@
                  "Plzensky": [
                               "L1PLMI01_T_N.csv.zip",
                              ],
+                 "Ustecky":[
+                            "U1SNEZ01_T_N.csv.zip",
+                            "U2VARN01_T_N.csv.zip"
```

Comment:
Díky za commit.
Malá drobnost. Za "U2VARN01_T_N.csv.zip" bude dobré dát čárku. Ten, kdo by "Ustecky" kraj doplňoval o další zdroje, by musel sáhnout do Tvé řádky a tu čárku tam přidat. Je přehlednější, aby do ní nesahal, když to není nutné a pokud by si toho nevšiml a nepřidal ji, tak by nám vznikla chyba.

Oprava je snadná:
* přidat čárku ve tvém kódu
* udělat další commit do stejné větve
* udělat push